umap/storage.py

```diff
@@ -38,34 +38,19 @@ class UmapManifestStaticFilesStorage(ManifestStaticFilesStorage):
         ),
     )
 
-    # https://github.com/django/django/blob/0fcee1676c7f14bb08e2cc662898dee56d9cf207↩
```

Comment:
This change is unclear to me.

Reply:
Previously, we were skipping `.map` files from the default regexp but now that these files are copied it's OK to get back to default and hash these filenames too.